src/handlers/admin/menu_handler.py

```diff
@@ -10,187 +11,103 @@
+        context.bot.edit_message_text(
+            text=constants.get_settings_message.format(**chat.__dict__),
+            parse_mode=ParseMode.MARKDOWN,
             chat_id=query.message.chat_id,
             message_id=query.message.message_id,
+            reply_markup=InlineKeyboardMarkup(keyboard),
         )
+    context.user_data["action"] = None
 
+# todo rework into callback folder
+def button_handler(update: Update, context: CallbackContext):
+    query = update.callback_query
+    data = json.loads(query.data)
+    
+    action_handlers = {
+        constants.Actions.start_select_chat: handle_start_select_chat,
+        constants.Actions.select_chat: handle_select_chat,
+        constants.Actions.get_current_settings: handle_get_current_settings,
+    }
+    
+    if data["action"] in action_handlers:
+        action_handlers[data["action"]](data, query, context)
+    
     elif data["action"] in [
         constants.Actions.set_on_new_chat_member_message_response,
         constants.Actions.set_kick_timeout,
-        constants.Actions.set_notify_message,
-        constants.Actions.set_on_known_new_chat_member_message_response,
-        constants.Actions.set_on_successful_introducion_response,
-        constants.Actions.set_on_kick_message,
+        # ... Other actions ...
     ]:
-        context.bot.edit_message_text(
-            text="Отправьте новое значение",
-            chat_id=query.message.chat_id,
-            message_id=query.message.message_id,
-        )
-        context.user_data["chat_id"] = data["chat_id"]
-        context.user_data["action"] = data["action"]
-
-    elif data["action"] == constants.Actions.get_current_settings:
-        keyboard = [
-            [
-                InlineKeyboardButton(
-                    "К настройке чата",
-                    callback_data=json.dumps(
-                        {
-                            "chat_id": data["chat_id"],
-                            "action": constants.Actions.select_chat,
-                        }
-                    ),
-                ),
-                InlineKeyboardButton(
-                    "К списку чатов",
-                    callback_data=json.dumps(
-                        {"action": constants.Actions.start_select_chat}
-                    ),
-                ),
-            ],
-        ]
-
-        reply_markup = InlineKeyboardMarkup(keyboard)
-        with session_scope() as sess:
-            chat = sess.query(Chat).filter(Chat.id == data["chat_id"]).first()
-            context.bot.edit_message_text(
-                text=constants.get_settings_message.format(**chat.__dict__),
-                parse_mode=ParseMode.MARKDOWN,
-                chat_id=query.message.chat_id,
-                message_id=query.message.message_id,
-                reply_markup=reply_markup,
-            )
-
-        context.user_data["action"] = None
-
+        handle_action(data, query, context)
+
+def set_chat_attribute(chat_id, attribute, value):
```

Comment:
anything which is not meant to be used outside of this module could start with `_` to be invisible for import suggestions